LICENSE

```diff
@@ -0,0 +1 @@
+TODO: Add your license here.
```

Comment:
@chemarubio, for the native SDKs we decided to not set explicitly any license as by doing that it is implied the most restricted one applies, should we do the same for this?

Reply:
Yes, you're right ! Since this file was auto-generated, I wanted it to be shared with the team, just in case we had the feeling some specific licence was needed to be in place for the Flutter plugin or not regarding its distribution.

I guess we should keep consistency among all tools, and I will remove this file 👍

Reply:
Wait, I had to add a license file to the iOS sdk, Cocoapods requirement. So you can use that if you wanted.